ofl/ibarrarealnova/OFL.txt

```diff
@@ -0,0 +1,88 @@
+Copyright (c) 2007 The Ibarra Real Authors.

```

Comment:
Should be

    Copyright (c) 2007 The Ibarra Real Nova Project Authors (https://github.com/googlefonts/ibarrareal)

Reply:
Given it is already merged, should I change it and restart the PR?

Reply:
Solved in [https://github.com/google/fonts/pull/2253](https://github.com/google/fonts/pull/2253)